src/main/resources/interface-specification.yml

```
@@ -3431,6 +3462,184 @@ components:
       schema:
         type: string
   schemas:
+    EServiceSeed:
+      type: object
+      required:
+        - producerId
+        - name
+        - description
+        - technology
+        - attributes
+      properties:
+        producerId:
+          type: string
+          format: uuid
+        name:
+          type: string
+          minLength: 5
+          maxLength: 60
+        description:
+          type: string
+          minLength: 10
+          maxLength: 250
```

constraints should not be in the BFF, we agreed to centralize them only on process services
```suggestion
        description:
          type: string
```